core/store/migrations/0034_webhook_external_initiator.go

```diff
@@ -0,0 +1,34 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up34 = `
+    ALTER TABLE external_initiators ADD CONSTRAINT external_initiators_name_unique UNIQUE(name);
```

Comment:
This is unnecessary, we already have a unique index on name

Reply:
No, Postgres balked at the FK until I did this

Reply:
Ohh... it's a functional index. And PG is not smart enough to realise that it already covers it.